Lib/sqlite3/dbapi2.py

```diff
@@ -25,6 +25,9 @@
 import collections.abc
 
 from _sqlite3 import *
+from _sqlite3 import _deprecated_version
+
+_deprecated_names = ["version", "version_info"]
```

Comment:
Minor: this could be a frozenset to save a bit of memory and speed up membership tests:
```suggestion
_deprecated_names = frozenset({"version", "version_info"})
```